### DISCUSSION TITLE
How to average the tetrahedral induced fields in the mesh file for each object and map them to the standard head model ernie.msh?

### DISCUSSION THREAD

**Discussion by xiongyy99255** [simnibs/simnibs#251] [9. Results and analysis]
Hi SimNIBS team, thanks for the great tool!
Is it possible to group average simNIBS .msh file, i.e. average the tetrahedra induced field of each object, and then map on a common ernie.msh template for visualization?
Thanks!
![example _average msh](https://github.com/simnibs/simnibs/assets/129670513/0945ce30-04c7-455c-af56-17dbeaef5d5d)

**Comment by TorgeW** [comment]
This is not possible but you can transform everything into mni space and work with volumetric regions of interest.
